managed-services/iot-hub/reference-content/certificate.mdx

```diff
@@ -0,0 +1,67 @@
+---
+meta:
+  title: Updating your Endpoint's Certificate Authority 
+  description: This page provides important information about the Scaleway IoT Hub Certificate Authority.
+content:
+  h1: Certificate Authority update
+  paragraph: This page provides important information about the Scaleway IoT Hub Certificate Authority.
+dates:
+  validation: 2023-04-27
+  posted: 2023-04-27
+categories:
+  - managed-services
+tags: iot iot-hub ca certificate authority update
+---
+
+As a security measure, Scaleway IoT Hub was designed with a mutual Transport Layer Security (TLS) as the default authentication protocol. This means each device needs both a certificate and key pair to connect to IoT Hub.
+
+Using a trusted Certificate Authority (CA) to issue digital certificates is essential to guarantee a secure Transport Layer Security (TLS) connection between a device and the IoT Hub. A digital certificate contains important information about the server's identify that can help verify its authenticity, then allowing the device to use it to establish a secure connection with the server.
+
+However, the current Certificate Authority of the IoT Hub server endpoint is set to expire **in February 2024**.
+
+The information provided on this page explains what the migration entails and how to successfully transition to the new CA.
+
+<Message type="note">
+  It is possible to replace the Scaleway-generated default certificate with either a self-generated or third-party certificate. If your devices do not use TLS, the information contained within this page will not pertain to you.
+</Message>
+
+The CA used in the migration is a `.pem` bundle that currently contains both the soon-to-be expired and new CA. This allows for a  seamless transition.
+
+# Planning the migration
+
+The new CA will now be valid for 10 years. This way, your devices will not need to be regularly updated. Additionally, the key used for the current CA has been changed from an RSA4096 to an ECDSA key with a NIST P-384 Curve. The new ECDSA algorithm uses shorter key lengths than RSA for stronger security, which is ideal for embedded devices.
+
+To implement these changes, a new IoT Hub endpoint has been made available: `migration.iot.fr-par-scw.cloud`. It acts exactly like `iot.fr-par-scw.cloud`, which is the endpoint Scaleway users have habitually been provided with. Starting February 2024, you can connect to this new, transitory endpoint.
```

Comment:
First there are typos in domain name it's not fr-par dash scw.cloud but fr-par DOT scw.cloud

This should mention that migration.iot.fr-par.scw.cloud uses a certificate signed with the new CA while iot.fr-par.scw.cloud still use the current CA.

The date is wrong as it's already available: 2023